lib/kino.ex

```diff
@@ -193,4 +212,20 @@ defmodule Kino do
   def nothing() do
     :"do not show this result in output"
   end
+
+  @doc """
+  Ties the given process lifetime to the caller.
+
+  When used directly in a Livebook cell, the process is killed
+  on re-evaluation.
+
+  When used from another process, the given process is killed
+  as soon as the parent terminates.
+  """
+  @spec bind_process(pid()) :: :ok | {:error, atom()}
+  def bind_process(pid) do
```

Comment:
Not sure if we want to expose this directly or only as `Kino.start_supervised` or `Kino.start_child` as in #51.